From: Ben Hutchings <ben@decadent.org.uk>
Subject: jbd2: Avoid ABI change in 3.2.45
Date: Fri, 07 Jun 2013 02:36:32 +0100

The enumeration for transaction_s::t_state was changed by commit
3464fc47e51e 'jbd2: fix race between jbd2_journal_remove_checkpoint
and ->j_commit_callback' in 3.2.45.  t_state is accessed only by jbd2
itself, not the filesystems that use jbd2, so this shouldn't affect
binary compatibility.  Hide the change from genksyms.
--- a/include/linux/jbd2.h
+++ b/include/linux/jbd2.h
@@ -470,7 +470,9 @@ struct transaction_s
 		T_COMMIT,
 		T_COMMIT_DFLUSH,
 		T_COMMIT_JFLUSH,
+#ifndef __GENKSYMS__
 		T_COMMIT_CALLBACK,
+#endif
 		T_FINISHED
 	}			t_state;
 
